src/model/enums/misc.rs

```diff
@@ -0,0 +1,105 @@
+use serde::{Deserialize, Serialize};
+use strum::{AsRefStr, Display, EnumString};
+
+/// disallow: interrupting_playback, pausing, resuming, seeking, skipping_next, skipping_prev, toggling_repeat_context, toggling_shuffle, toggling_repeat_track, transferring_playback
+#[derive(
+    Clone, Serialize, Deserialize, Copy, PartialEq, Eq, Debug, Hash, EnumString, AsRefStr, Display,
```

Comment:
May I ask why we use `AsRefStr` instead of `ToString` for these derive macros? What is the `disallow: ...` part in the comment as well?